packages/frontend/src/utils/tokens.js

```diff
@@ -5,30 +5,21 @@ import sendJson from '../tmp_fetch_send_json';
 import { ACCOUNT_HELPER_URL } from './wallet';
```

Comment:
This file should be deleted rather than modified -- it has been _replaced_ with `FungibleTokens` service https://github.com/near/near-wallet/blob/cffe2d4829ffb9c42efe4853ae2b5a1609b64a75/packages/frontend/src/services/FungibleTokens.js#L44, whose associated functions already have the correct signature and return values.

Reply:
Done.